boards/jiminy-mega256rfr2/board.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2016 RWTH Aachen, Josua Arndt
+ * Copyright (C) 2014 Freie Universität Berlin, Hinnerk van Bruinehsen
```

Comment:
I don`t think there is an other author then we for this file! Nobody else will answer question related to this file then we.